server/sonar-web/src/main/js/apps/issues/sidebar/AssigneeFacet.js

```diff
@@ -79,7 +79,7 @@ export default class AssigneeFacet extends React.PureComponent {
   handleSearch = (query /*: string */) => {
     let organization = this.props.component && this.props.component.organization;
     if (this.props.organization && !organization) {
-      organization = this.props.organization.key;
+      ({ organization } = this.props);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract the assignment of "{ organization }" from this expression. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)